examples/02-plot/bounds.py

```diff
@@ -0,0 +1,72 @@
+"""
+.. _bounds_example:
+
+Plotting Bounds
+~~~~~~~~~~~~~~~
+
+This example demonstrates to show bounds within a :class:`pyvista.Plotter`
+using :func:`show_grid() <pyvista.Plotter.show_grid>`
+
+"""
+# sphinx_gallery_thumbnail_number = 2
+
+import pyvista as pv
+from pyvista import examples
+
+###############################################################################
+# Show All Bounds
+# ~~~~~~~~~~~~~~~
+# In this plot we show the bounds for all axes by setting ``location='all'``.
+
+plotter = pv.Plotter()
+plotter.add_mesh(pv.Sphere(), smooth_shading=True)
+plotter.show_bounds(location='all')
+plotter.show()
+
+
+###############################################################################
+# Override Shown Values Limits
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# In this example we override the values shown along the X and Y axes with our
+# own custom values. This can be useful when you wish to display different
+# values along the axes without having to scale the dataset. Also note how we
+# hide the Z labels by setting ``show_zlabels=False``.
+
+gears = examples.download_gears()
+
+plotter = pv.Plotter()
+plotter.add_mesh(gears, smooth_shading=True, split_sharp_edges=True)
+plotter.show_bounds(axes_ranges=[0, 5, 0, 5, 0, 2], show_zlabels=False)
+plotter.show()
+
```

Comment:
I assume this is a VTK thing, but this is what the plot looks like:
![Screenshot from 2022-06-18 15-24-53](https://user-images.githubusercontent.com/17914410/174439732-53d12cd7-f7fc-4f34-b17e-67a4db8a6089.png)
note the arrows on the axis lines pointing from higher values toward lower ones.

Reply:
I think this is a VTK “feature”. Reversing the range doesn’t work either (e.g. manually providing reversed bounds).

Reply:
Nevermind, I'm dumb. Those are not arrows, those are ticks plotted in the xy/xz/yz planes... they just look bad.